comps/comps-katello-client-fedora27.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+
+<comps>
+<!--  <meta> -->
+<!-- Meta information will go here eventually -->
+<!--  </meta> -->
+  <group>
+    <id>katello-client</id>
+    <name>Katello Client</name>
+    <description>Katello Client Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+       <packagereq type="default">katello-agent</packagereq>
+       <packagereq type="default">katello-host-tools</packagereq>
+       <packagereq type="default">katello-host-tools-fact-plugin</packagereq>
+       <packagereq type="default">katello-client-repos</packagereq>
```

Comment:
Can you add `katello-host-tools-tracer` ?

Reply:
sure